src/routes/learn/distribution.svx

```diff
@@ -0,0 +1,176 @@
+---
+title: Distribution and Histograms
+blurb: |
+    Histograms, distribution functions, and the normal distribution
+tags: 
+    - distribution
+    - standard deviation
+    - mean
+    - median
+    - mode
+    - normal
+    - histogram
+    - stats
+    - statistics
+flair: tutorial
+---
```

Comment:
featured image needs to be specified here.

see https://github.com/flucoma/learn-website/blob/d587fc0f802c26978062109e1b7c45e9f0010cef/src/routes/learn/bufnmf.svx#L11 for an example